### PR TITLE
DOC-1581: Updated support pages

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: DOC-1581

Description of Changes:
* Updated support page
* Update the supported premium plugins table
* Removed any Internet Explorer stuffs in the powerpaste troubleshoot
* Remove any mentions to Internet Explorer in the `browser_spellcheck` and `fullscreen_native` doc

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Comment by tiny-ben-tran** [comment]
There are some Internet Explorer related stuffs on this [page](https://www.staging.tiny.cloud/docs/tinymce/6/troubleshoot/#usingbrowsertoolingtoinvestigateservicesissues). It seems to relate to server configuration so I leave them out as they are not in the scope of this ticket.

File: modules/ROOT/pages/powerpaste-troubleshooting.adoc [hunk 1]
```diff
@@ -20,18 +20,4 @@ Microsoft Word can also create content that cannot be accurately recreated in HT
...
-
-== Why do images not paste when copied with text content in Microsoft Internet Explorer 11?
-
-This issue relates to changes to Microsoft Internet Explorer 11 late in the product’s life. {companyname} (and companies that offer similar products) have reached out to Microsoft to suggest that this change is a defect despite their initial reply that it was intentional and "expected behavior" in Microsoft Internet Explorer 11. Microsoft has made no public statement about addressing the issue specifically, and is no longer making non-security changes to Microsoft Internet Explorer 11. The only recommended workarounds are:
-
-* Paste the images into {productname} individually.
-* Use a different browser.
+To view the HTML of content pasted from the clipboard: http://static.ephox.com/clipboard/clipboardtest.html[{companyname} Clipboard Viewer].
```
Comment (lnewson) [inline modules/ROOT/pages/powerpaste-troubleshooting.adoc:23]: This doesn't read well... it says to view it and then just has a link (granted this was sort of a problem before as well). Maybe something like this?
```suggestion
To view the HTML of content on the clipboard paste into the red box on the http://static.ephox.com/clipboard/clipboardtest.html[{companyname} Clipboard Viewer].
```

File: modules/ROOT/partials/misc/plugin_support_table.adoc [hunk 2]
```diff
@@ -21,3 +21,5 @@ The following list of premium plugins are supported by {companynameformal} for {
 * xref:rtc-introduction.adoc[Real-Time Collaboration (RTC)]
 * xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro]
 * xref:premium-skins-and-icons.adoc[{prem_skins_icons}]
+* xref:editimage.adoc[EditImage]
+* xref:tableofcontents.adoc[TableOfContents]
```
Comment (lnewson) [inline modules/ROOT/partials/misc/plugin_support_table.adoc:25]: These need to be in alphabetical order above the premium skins and icon packs.

File: modules/ROOT/partials/misc/supported_versions.adoc [hunk 3]
```diff
@@ -6,14 +6,7 @@ Supported versions of {productname}:
...
 |Version |Release Date |End of Support
-|6.0 |TBD |TBD
-|5.9 |September 15, 2021 |TBD
-|5.8 |May 19, 2021 |November 19, 2022
-|5.7 |February 24, 2021 |August 24, 2022
-|5.6 |December 8, 2020 |June 8, 2022
-|5.5 |October 21, 2020 |April 21, 2022
-|5.4 |July 23, 2020 |January 23, 2022
-|5.3 |June 11, 2020 |December 11, 2021
+|6.0 |April 06, 2022 |TBD
```
Comment (lnewson) [inline modules/ROOT/partials/misc/supported_versions.adoc:9]: We probably should leave this as TBD now, especially if we're doing another prod push later this week as we don't want a fixed date advertised yet.

File: modules/ROOT/partials/configuration/browser_spellcheck.adoc [hunk 4]
```diff
@@ -3,8 +3,6 @@
 
 This option configures {productname} to use the browser's native spell checker.
 
-`+browser_spellcheck+` is not available in either Internet Explorer 8 or 9, as neither browser has native spell checker functionality. For more complete browser support, we recommend using xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro].
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/browser_spellcheck.adoc:6]: We probably don't want to lose the last sentence here? As we'd want to mention/highlight that Spell Checker Pro works better than native.

**Review by lnewson — CHANGES_REQUESTED** [review]
Looks good overall, just found a few smaller nuances 👍

File: modules/ROOT/partials/configuration/browser_spellcheck.adoc (hunk 4 above)
Comment (tiny-ben-tran) [reply modules/ROOT/partials/configuration/browser_spellcheck.adoc:6]: The last sentence needs to be reworded a bit.

How about `For better spell checking functionality, we recommend using xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro]`

Comment (brian-forte) [reply modules/ROOT/partials/configuration/browser_spellcheck.adoc:6]: For integrated spell-checking, we recommend xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro].

File: modules/ROOT/partials/misc/plugin_support_table.adoc [hunk 5]
```diff
@@ -9,6 +9,7 @@ The following list of premium plugins are supported by {companynameformal} for {
 * xref:casechange.adoc[Case Change]
 * xref:checklist.adoc[Checklist]
 * xref:introduction-to-tiny-comments.adoc[Comments]
+* xref:editimage.adoc[EditImage]
```
Comment (lnewson) [inline modules/ROOT/partials/misc/plugin_support_table.adoc:12]:
```suggestion
* xref:editimage.adoc[Edit Image]
```
This name is incorrect, it should be this.

File: modules/ROOT/partials/misc/plugin_support_table.adoc [hunk 6]
```diff
@@ -20,6 +21,5 @@ The following list of premium plugins are supported by {companynameformal} for {
 * xref:introduction-to-powerpaste.adoc[PowerPaste]
 * xref:rtc-introduction.adoc[Real-Time Collaboration (RTC)]
 * xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro]
-* xref:premium-skins-and-icons.adoc[{prem_skins_icons}]
-* xref:editimage.adoc[EditImage]
-* xref:tableofcontents.adoc[TableOfContents]
\ No newline at end of file
+* xref:tableofcontents.adoc[TableOfContents]
```
Comment (lnewson) [inline modules/ROOT/partials/misc/plugin_support_table.adoc:24]:
```suggestion
* xref:tableofcontents.adoc[Table of Contents]
```
This name is incorrect, it should be this.

File: modules/ROOT/partials/configuration/browser_spellcheck.adoc (hunk 4 above)
Comment (lnewson) [reply modules/ROOT/partials/configuration/browser_spellcheck.adoc:6]: `we` is a pronoun so isn't something we should use for technical documentation... see https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns. Maybe something like this instead:
```suggestion
For integrated spell-checking, checkout the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin.
```

**Review by lnewson — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/configuration/browser_spellcheck.adoc (hunk 4 above)
Comment (lnewson) [reply modules/ROOT/partials/configuration/browser_spellcheck.adoc:6]: Another option would be to substitute `we` with `{companyname}` but I don't think that's the right solution here.

File: modules/ROOT/partials/misc/plugin_support_table.adoc (hunk 6 above)
Comment (tiny-ben-tran) [reply modules/ROOT/partials/misc/plugin_support_table.adoc:24]: Just curious, are the name of MoxieManager and PowerPaste correct?

Comment (lnewson) [reply modules/ROOT/partials/misc/plugin_support_table.adoc:24]: Yes they are correct. Take a look at the actual plugin pages to see their name, or look at the product pages. For example: https://www.tiny.cloud/tinymce/features/powerpaste

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/configuration/fullscreen_native.adoc [hunk 7]
```diff
@@ -33,5 +33,4 @@ The `+fullscreen_native+` option has the following limitations. When `+fullscree
 
 * The Escape keyboard key (Esc) will exit full screen mode. Default editor behavior of the Esc key will be overridden by browser and the Esc key will exit full screen mode instead of closing dialogs, menus, or moving focus from the editor UI to the editor content. This may lead to accessibility issues.
 * Firefox users - The full screen keyboard shortcut (Ctrl+Shift+F) cannot be used to repeatedly toggle the browser-native full screen mode on and off without interacting with the editor between on toggles. If the user does not interact with the editor between off and on toggles, the plugin will fall-back to filling the browser window.
-* Microsoft Internet Explorer 11 users - The full screen keyboard shortcut (Ctrl+Shift+F) will not activate the browser-native full screen. If the full screen shortcut is used, the plugin will fall-back to filling the browser window.
-* If the editor is initialized inside an iframe element, full screen mode will only fill the iframe on Firefox and Microsoft Internet Explorer 11.
+* If the editor is initialized inside an iframe element, full screen mode will only fill the iframe on Firefox
```
Comment (metricjs) [inline modules/ROOT/partials/configuration/fullscreen_native.adoc:36]:
```suggestion
* If the editor is initialized inside an iframe element, full screen mode will only fill the iframe on Firefox.
```

**Review by metricjs — APPROVED** [review]
Approving provided open comments are resolved

**Review by TheSpyder — APPROVED** [review]

**Pull request merged**